### PR TITLE
Dropdown menu fix autoscroll carrousel

### PR DESCRIPTION
**Pull request by JaqKent**

### CODE REVIEW

File: src/components/Slider/index.tsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faChevronLeft,
+  faChevronRight,
+} from '@fortawesome/free-solid-svg-icons';
+
+import styles from './styles.module.scss';
+
+interface Props {
+  slides: any;
```
Comment (nathan-hyan) [inline src/components/Slider/index.tsx:15]: Replace Any

File: src/components/Slider/index.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+  auto ? (useEffect(() => {
+    const timeOut = autoPlay ? setTimeout(() => {
+      nextSlide();
+    }, 2500) : undefined;
+
+    return () => {
+      clearTimeout(timeOut);
+    };
+  })) : '';
```
Comment (nathan-hyan) [inline src/components/Slider/index.tsx:34]:
```suggestion
useEffect(() => {
    const timeOut = auto ? setTimeout(() => {
      nextSlide();
    }, 2500) : undefined;

    return () => {
      clearTimeout(timeOut);
    };
  })
```

File: src/reportWebVitals.ts [hunk 3]
```diff
@@ -2,7 +2,9 @@ import { ReportHandler } from 'web-vitals';
 
 const reportWebVitals = (onPerfEntry?: ReportHandler) => {
   if (onPerfEntry && onPerfEntry instanceof Function) {
-    import('web-vitals').then(({ getCLS, getFID, getFCP, getLCP, getTTFB }) => {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
       getCLS(onPerfEntry);
       getFID(onPerfEntry);
       getFCP(onPerfEntry);
```
Comment (nathan-hyan) [inline src/reportWebVitals.ts:10]: if u want, delete this file

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
great job

File: src/components/CustomButton/index.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/button-has-type */
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/CustomButton/index.tsx:1]: done

File: src/components/CustomButton/index.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    secondary?:boolean;
+    small?: boolean;
+}
+
+function CustomButton({
+  label, onClick, className, small, isSubmit, disabled, secondary,
+}: Props) {
+  return (
+    <div className={className}>
+      <button
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/CustomButton/index.tsx:19]: done

File: src/components/CustomButton/styles.module.scss [hunk 6]
```diff
@@ -0,0 +1,61 @@
+@import 'scss/colors';
+@import 'scss/mixins';
+@import 'scss/variable';
+
+
+
+
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/CustomButton/styles.module.scss:7]: done

File: src/components/NavBar/DropdownMenu/LogedinFront/styles.module.scss [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+        border-radius: 50px ;
+        height: 4em;
+        margin-left: 2rem;
+        width: 4em;
+    }
+    h3{
+        font-size: 1.5em;
+    }
+  
+
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/NavBar/DropdownMenu/LogedinFront/styles.module.scss:24]: done

File: src/components/NavBar/DropdownMenu/index.tsx [hunk 8]
```diff
@@ -1,65 +1,66 @@
...
 
-function DropsdownMenu() {
+function DropdownMenu() {
+  const [logged, setLogged] = useState(false);
+
   const handleLink = () => {
     console.log('links');
   };
   const handleLogin = () => {
     console.log('login');
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/NavBar/DropdownMenu/index.tsx:16]: done

File: src/components/NavBar/DropdownMenu/styles.module.scss [hunk 9]
```diff
@@ -4,12 +4,23 @@
...
+  hr{
+    margin-left: 2em;
+    width: 30rem;
+  }
+  .button{
+    @include flex(space-around, center, row);
+    padding: 0.5em 0 0 2em;
+  }
+  .links{
+   padding-left: 2em;
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/NavBar/DropdownMenu/styles.module.scss:22]: em ;)

File: src/components/NavBar/DropdownMenu/utils.ts [hunk 10]
```diff
@@ -0,0 +1,20 @@
+export const categoryId = (id: string | undefined) => {
+  switch (id) {
```
[the remark this answers is not in the document]
Comment (JaqKent) [reply src/components/NavBar/DropdownMenu/utils.ts:2]: done

File: src/components/Slider/index.tsx (hunk 1 above)
Comment (JaqKent) [reply src/components/Slider/index.tsx:15]: done

**Pull request merged**